Libraries/DirectMLX.h

```diff
@@ -2342,7 +2341,14 @@ namespace dml
             axisSizeSum += outputAxisSize;
```

Comment:
Won't this now fail to compile?

Reply:
Concur, as I see `axisSizeSum` being used on line 2342.

Reply:
Issue #220 describes the detailed error log. The`axisSizeSum` used in 2342 only be assignment for the newer clang. The  `axisSizeSum` only be used for `assert` statement on line 2345.

Reply:
Line 2341 should be deleted now.